packages/loopring_v3/contracts/impl/ProtocolFeeVault.sol

```diff
@@ -100,169 +75,85 @@ contract ProtocolFeeVault is Claimable, ReentrancyGuard, IProtocolFeeVault
         external
         nonReentrant
     {
-        require(msg.sender == userStakingPoolAddress, "UNAUTHORIZED");
-
         require(
-            lrcAddress.safeTransfer(msg.sender, amount),
-            "TRANSFER_FAILURE"
+            userStakingPoolAddress != address(0) &&
+            msg.sender == userStakingPoolAddress,
```

Comment:
`msg.sender` can't be `0x0` so the extra `userStakingPoolAddress != address(0)` check seems unnecessary.

Reply:
fixed